GovUkDesignSystemComponents/SubComponents/SummaryListRowAction.cshtml

```diff
@@ -0,0 +1,18 @@
+﻿@model GovUkDesignSystem.GovUkDesignSystemComponents.SummaryListRowActionViewModel
+
+<dd class="govuk-summary-list__actions @Model?.Classes">
+    @{
+        if (Model != null && Model.Items != null && Model.Items.Count > 0)
+        {
+            for (var index = 0; index < Model.Items.Count; index++)
```

Comment:
Sorry, didn't spot this earlier, I think this would be quite a lot neater with a foreach

Reply:
Sorry, I should have spotted it in first review changes. Done, thanks